Gemfile

```diff
@@ -11,3 +11,4 @@ gem "yard"
 gem "rubocop", ">= 1.0"
 gem "rubocop-shopify", require: false
 gem "benchmark-ips"
+gem "mutex_m"
```

Comment:
Please don't use `mutex_m` it's pointless and expose public methods. Just use a simple mutex as an ivar.

Reply:
Yeah, you need to update minitest.